src/components/cards/next-event-card/index.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, Flex, Text } from "@radix-ui/themes";
+import Image from "next/image";
+import Link from "next/link";
+
+import { INextEvent, padToStartNumber } from "@/src/constants/next-events-mock";
+
+export const NextEventCard: ({
+  type,
+  title,
+  link,
+  date,
+  img,
+  time,
+}: INextEvent) => JSX.Element = ({
+  type,
+  title,
+  link,
+  date,
+  img,
+  time,
+}: INextEvent) => {
+  function formatDateWithDayName(date: Date): string {
+    const daysOfWeek: string[] = [
+      "Sunday",
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+    ];
+    const dayName: string = daysOfWeek[date.getDay()];
+
+    const day: string = String(date.getDate()).padStart(padToStartNumber, "0");
+    const month: string = String(date.getMonth() + 1).padStart(
+      padToStartNumber,
+      "0"
+    );
+    const year: number = date.getFullYear();
+
+    return `${dayName}, ${day}.${month}.${year}`;
+  }
+
+  return (
+    <Box className="max-w-[310px] w-full border border-solid border-black  bg-white">
+      <Box className="w-full h-[228px]">
+        <Image src={`${img}`} alt="image" height={228} width={310} />
+      </Box>
+
+      <Box className="py-[17px] px-[20px]">
+        <Text as="p" className="font-medium uppercase text-dark-red text-sm">
+          {type}
+        </Text>
+
+        <Text
+          as="p"
+          className="mt-2 font-medium text-xl h-20 line-clamp-3 text-ellipsis overflow-hidden ..."
```

Comment:
what is `...` means?

Reply:
its line-break in case title is too big. So every title won't be bigger than 3 lines